### ISSUE TITLE
No burndown chart visible when selecting multiple labels

### ISSUE THREAD

**Issue by adamreisnz** [ZenHubIO/support#1301]
![image](https://user-images.githubusercontent.com/490562/32984785-fab20846-cd11-11e7-9015-c79b5595bfcd.png)

When I select the labels one by one, I can see a burndown chart for each label, but when I select both, it doesn't show anything.

**Comment by kendramoroz** [comment]
Hey @adamreisnz thanks for sending this through! The filters work using the `and` sequence, not `or` - this means this new filter combination is looking for Issues that have both labels, `bug` and `test` - can you confirm if there's any estimated Issues in this Milestone with both of those labels on them?

**Comment by adamreisnz** [comment]
No there aren't any issues like that, but that's also unlikely to happen. What if I want to know the combined burndown for both `bug` and `test` issues?

**Comment by kendramoroz** [comment]
agreed 👍 - we're going to be making improvements to reporting workflows so we'll keep this issued up to dig further into.
